packages/manager/modules/account-sidebar/src/user-infos/template.html

```diff
@@ -82,10 +82,16 @@
         </p>
     </div>
     <div
-        class="text-left pb-2 manager-hub-user-infos_links"
+        class="text-left manager-hub-user-infos_links"
```

Comment:
It's not related to this Pull Request but it's strange to have a class
that refer to the Hub application from the `account-sidebar` module.

Reply:
You're right, the hub sidebar has moved to account-sidebar module but the class names have not been touched. I'll renamed the classes when the remaining PRs on account sidebar will be merged

Reply:
@antleblanc Thanks for pointing this out, as @frenautvh said, it will be taken care of with some refactoring later on